.travis.yml

```diff
@@ -204,7 +204,7 @@ jobs:
         # Set MAVEN_OPTS for Surefire launcher. Skip remoteresources to avoid intermittent connection timeouts when
         # resolving the SIGAR dependency.
         - >
-          MAVEN_OPTS='-Xmx1100m' ${MVN} test -pl ${MAVEN_PROJECTS}
+          MAVEN_OPTS='-Xmx2048m' ${MVN} test -pl ${MAVEN_PROJECTS}
```

Comment:
was there a reason related to the plugin update that we needed the extra memory

Reply:
looks like https://app.travis-ci.com/github/apache/druid/jobs/565818329#L1410